stacker/providers/aws/default.py

```diff
@@ -129,13 +170,21 @@ def is_stack_destroyed(self, stack, **kwargs):
         return self.get_stack_status(stack) == self.DELETED_STATUS
 
     def tail_stack(self, stack, retries=0, **kwargs):
-        def log_func(e):
-            event_args = [e['ResourceStatus'], e['ResourceType'],
-                          e.get('ResourceStatusReason', None)]
-            # filter out any values that are empty
-            event_args = [arg for arg in event_args if arg]
-            template = " ".join(["[%s]"] + ["%s" for _ in event_args])
-            logger.info(template, *([stack.fqn] + event_args))
+        def log_func(message):
+            # Check that the message is for the correct stack
+            if message.StackName == stack.fqn:
```

Comment:
Is this actually necessary anymore?  I think you'd only check the stack name if you were tailing the stack the old way?  Maybe this is partially due to the confusion introduced by keeping the old methodology.